security/src/main/java/pm/gnosis/svalinn/security/impls/AndroidFingerprintHelper.kt

```diff
@@ -51,10 +54,10 @@ class AndroidFingerprintHelper(private val context: Context) : FingerprintHelper
         return keyStore.getKey(FINGERPRINT_KEY, null) as? SecretKey ?: createKey()
     }
 
-    override fun isKeySet(): Single<Boolean> = Single.fromCallable {
+    override fun isKeySet(): Boolean {
```

Comment:
`suspend` ? Sorry again, feel free to just resolve all these sort of comments. I am leaving these as sort of `bookmarks` for you.